src/unicode.cpp

```diff
@@ -60,22 +60,20 @@ ada_really_inline bool has_tabs_or_newline(
   }
   // fast path for long strings (expected to be common)
   size_t i = 0;
-  const uint8x16_t mask1 = vmovq_n_u8('\r');
-  const uint8x16_t mask2 = vmovq_n_u8('\n');
-  const uint8x16_t mask3 = vmovq_n_u8('\t');
+  static uint8_t rnt_array[16] = {1, 0,  0, 0, 0,  0, 0, 0,
```

Comment:
Aleternatively:
a) can you just allocate the url aligned to 64 bytes or smth?
b) How big are urls in general? I presume between 8 and 16 bytes is also common? Could maybe 8 byte arm version be helpful?

Reply:
We have no control over how the string was allocated. It is passed to us by the user.